### ISSUE TITLE
use buildroot instead of sid

### ISSUE THREAD

**Issue by rbucker** [coreos/rkt-builder#2]
I know that this project is meant to be a "we JUST need to build rkt", however, I'm also using the project as a base image for my build oneliners. My concern is that my image is 1.4G in size.

```
coreos.com/rkt/builder:1.0.1                            2 weeks ago     2 weeks ago     1.4GiB  false
```

After watching a redbeard [demo](https://www.youtube.com/watch?v=gMpldbcMHuI) that included "building minimal distros". I would have thought that would have been the plan.

(I would have attached this request to the "trim" enhancement but this request is bigger than a breadbox. I'll bet redbeard could be a good resource)

**Comment by s-urbaniak** [comment]
Hi,

Actually we wanted to use buildroot initially. I had a few iterations with redbeard, but we both came to the conclusion, that buildroot is not the best option for this use case.

buildroot actually has no option to include gcc and go (a build env) on the target, which is our primary use case for a builder (they had preliminary support for gcc on the target in the past, but removed that feature). redbeard uses the gentoo stage3 image to provide a containerized build environment (https://github.com/brianredbeard/minimal_containers#aci) but suggested in an OOB discussion that the final size will not differ much from what we use here with Debian.

I do agree that the deflated image is pretty big right now, but there is potential to strip the current image as indicated in #1 and I'll work on that. Given that this image does not change very often, and having a still manageable download size (240MB) we decided to stick with the Debian environment.

PS: I built a builder buildroot image for go-based projects without gcc though, if that is of interest I can push a small repo with the buildroot configuration.

**Comment by rbucker** [comment]
Thanks for the history lesson. My reason for creating this tkt was because I make some recommendations to the acbuild, docker2aci, projects that they use rkt to build so that the build process could be initiated from the CoreOS host using rkt. My initial image was alpine linux but I was scolded for using a large image, Then I found the found the redbeard video. Then I determined that I was building large images in my CI and I was drinking huge amounts of coolaid.

Thanks again.
